Sources/Transcribe/WebVTT.swift

```diff
@@ -5,6 +5,7 @@ extension AmazonTranscribe.Transcript {
     public func makeWebVTT() -> String {
 
         let formatter = DateFormatter()
+        formatter.locale = Locale(identifier: "en_US_POSIX")
```

Comment:
@JulianKahnert This is the only significant code change I made. Using this fixed locale is important in case other calendars use different representations for seconds/minutes/hours (although I'm not aware of any common calendar that does so).

Reply:
@ole thanks for having a look at the code! You are right. It should be a fixes Locale.